edalize/symbiflow.py

```diff
@@ -148,6 +148,14 @@ def configure_nextpnr(self):
                                  'vivado.sh',
                                  dict())
 
+        vendor = self.tool_options.get('vendor', None)
+
+
+        # Optional script that will be sourced right before executing each build step in Makefile
+        # This script can for example setup enviroment variables or conda enviroment.
+        # This file needs to be a bash file
+        environment_script = self.tool_options.get('environment_script', None)
```

Comment:
A reduced version of this comment should end up also here: https://github.com/SymbiFlow/edalize/blob/1a16ecbf8f0b855e9a35279c7c4a66c6c3661801/edalize/symbiflow.py#L56-L62

Reply:
OK, added.